swath_projector/earthdata_varinfo_config.json

```diff
@@ -21,6 +44,20 @@
         }
       ],
       "_Description": "VNP10 SnowData variables have incorrect relative paths for coordinates."
+    },
+    {
+      "Applicability": {
+        "Mission": "TEMPO",
+        "ShortNamePath": "TEMPO_O3TOT_L2",
+        "VariablePattern": "^/product/.*|^/support_data/.*"
+      },
+      "Attributes": [
+        {
+          "Name": "coordinates",
+          "Value": "/geolocation/latitude, /geolocation/longitude"
+        }
+      ],
+      "_Description": "TEMPO_O3TOT_L2 variables only contain basenames for coordinates, which are found in sibling hierarchical groups. This rule fully qualifies the paths to these coordinates."
```

Comment:
Nit: Just for clarity it might be worth mentioning in this description that some variables in these groups are excluded via `ExcludedScienceVariables`. (Just to link the two parts of the configuration file solution together)

Reply:
Added wording in commit [6678d5a](https://github.com/nasa/harmony-swath-projector/pull/19/commits/6678d5a8f93fab5a79213d7ceb16a13538fb52c1)